.github/workflows/maven-ci.yml

```diff
@@ -0,0 +1,98 @@
+name: OTP CI Build
+
+on:
+  push:
+  pull_request:
+    branches:
+      - ibi-dev
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_PROFILE: default
+      AWS_REGION: us-east-1
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      MAVEN_OPTS: -Xmx2g
```

Comment:
This makes available our AWS access keys to all steps and thus third party applications. If possible, can these be included only in the individual steps that need this logic?

Reply:
AWS creds moved into individual step.